### PR TITLE
Update dependencies to support Hapi v21

### PR DESCRIPTION
**Pull request by rossanthony**
Addresses issue: https://github.com/pinojs/hapi-pino/issues/179

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,13 +4,19 @@
...
+| v2.x          | v16           | v4            |
 
-- hapi-pino v9.x supports Hapi v20.
-- hapi-pino v8.x supports Hapi v18, v19 and v20.
-- hapi-pino v7.x supports Hapi v18 and v19.
-- hapi-pino v6.x supports Hapi v17, v18 and v19
-- hapi-pino v5.x supports Hapi v17 and v18
 - hapi-pino v3.x
 supports Hapi v17 only. The maximum version that can be used with Hapi v16 is Pino v4.
 - [hapi-pino v2.x](https://github.com/pinojs/hapi-pino/tree/v2.x.x) is the LTS line for Hapi v16.
```
Comment (mcollina) [inline README.md:22]: Please remove those lines

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "hapi-pino",
-  "version": "11.0.0",
+  "version": "12.0.0",
```
Comment (mcollina) [inline package.json:3]: I'll bump this on release, please revert

Comment (rossanthony) [reply package.json]: removed

File: README.md (hunk 1 above)
Comment (rossanthony) [reply README.md]: removed

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**